styles/modules.css

```diff
@@ -1,15 +1,5 @@
-.clearfix {
-  clear: both;
-}
-
 body {
-  background: linear-gradient( 45deg, yellow, orange, white);
```

Comment:
You made good use of the gradient styling, but not sure the bright color and gradient work well particularly when going from narrow to wide viewports.  Its really cool, but possibly make use of it in a smaller context than the entire page background.